examples/smallvil/state.rs

```diff
@@ -0,0 +1,153 @@
+use std::{os::unix::prelude::AsRawFd, sync::Arc};
+
+use calloop::{generic::Generic, EventLoop, Interest, LoopSignal, Mode, PostAction};
+use slog::Logger;
+use smithay::{
+    desktop::{Space, WindowSurfaceType},
+    utils::{Logical, Point},
+    wayland::{
+        compositor::CompositorState,
+        output::OutputManagerState,
+        seat::{Seat, SeatState},
+        shell::xdg::XdgShellState,
+        shm::ShmState,
+        socket::ListeningSocketSource,
+    },
+};
+use wayland_server::{
+    backend::{ClientData, ClientId, DisconnectReason},
+    protocol::wl_surface::WlSurface,
+    Client, Display,
+};
+
+use crate::CalloopData;
+
+pub struct Smallvil {
+    pub pointer_location: Point<f64, Logical>,
+
+    pub start_time: std::time::Instant,
+    pub clients: Vec<Client>,
```

Comment:
I believe this field isn't used any more? At least from what I see with the listening socket.

Reply:
right